kld/tests/smoke/start.rs

```diff
@@ -181,9 +182,10 @@ pub async fn test_start() -> Result<()> {
     Ok(())
 }
 
+#[serial_test::serial]
+#[test_with::no_env(TEST_IN_FLAKE)]
 #[tokio::test(flavor = "multi_thread")]
-#[ignore = "Only run this for manual testing"]
-pub async fn test_manual() -> Result<()> {
+pub async fn test_stability() -> Result<()> {
```

Comment:
Why have you changed this? What is the intention now? If the ignored test really bothers you then just delete it I edit test_start to do manual testing.

Reply:
Just change the name to the exact thing it checks on.  If I am not misunderstanding, it tests on the stability of kld by running 10000 seconds, not something called `manual` (manual is the way we run it, not the subject of test).  I can change this back if the naming of the test does not matter.